src/Text/Pandoc/Error.hs

```diff
@@ -124,6 +124,8 @@ renderError e =
                  "\nTry using Word to save your DOC file as DOCX," <>
                  " and convert that with pandoc."
         "pdf" -> "\nPandoc can convert to PDF, but not from PDF."
+        "asciidoc" -> "\nPandoc can convert to asciidoc, but not from " <>
+                      "\n asciidoc (sometimes adoc also)."
```

Comment:
I'd prefer a simpler one liner "Pandoc can convert to asciidoc, but not from asciidoc."

Reply:
@jgm done